frontend/src/views/Share.vue

```diff
@@ -93,7 +93,7 @@
             </a>
             <a
               target="_blank"
-              :href="link + '?inline=true'"
+              :href="link + '&inline=true'"
```

Comment:
This will make the link invalid on shares without password.

Reply:
Good point, I really need to properly construct the query string using official builder methods.